server/src/server.ts

```diff
@@ -375,19 +376,37 @@ function onMessage(msg: m.Message) {
         // error: code and message set in case an exception happens during the definition request.
       };
       send(definitionResponse);
+    } else if (msg.method === p.ReferencesRequest.method) {
+      // https://microsoft.github.io/language-server-protocol/specifications/specification-current/#textDocument_references
+      let result: Location | null = utils.runAnalysisAfterSanityCheck(
```

Comment:
The annotation here is wrong btw. It should be `Location[]`

Reply:
I should put a comment somewhere, but basically `utils.runAnalysisAfterSanityCheck` returns `any` since it decodes some JSON. We don't carry the `result`s around but I wanted to annotate them just in case and for doc.

Reply:
I got to used to types actually meaning something, as opposed to be room decoration.

Reply:
done